app/ante/handlers.go

```diff
@@ -0,0 +1,93 @@
+package ante
+
+import (
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/ante"
+	authante "github.com/cosmos/cosmos-sdk/x/auth/ante"
+	vestingtypes "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+	ibcante "github.com/cosmos/ibc-go/v6/modules/core/ante"
+	ethante "github.com/evmos/ethermint/app/ante"
+)
+
+// NOTE: this function is copied from ethermint
+func newEthAnteHandler(options HandlerOptions) sdk.AnteHandler {
+	return sdk.ChainAnteDecorators(
+		ethante.NewEthSetUpContextDecorator(options.EvmKeeper),                         // outermost AnteDecorator. SetUpContext must be called first
+		ethante.NewEthMempoolFeeDecorator(options.EvmKeeper),                           // Check eth effective gas price against minimal-gas-prices
+		ethante.NewEthMinGasPriceDecorator(options.FeeMarketKeeper, options.EvmKeeper), // Check eth effective gas price against the global MinGasPrice
+		ethante.NewEthValidateBasicDecorator(options.EvmKeeper),
+		ethante.NewEthSigVerificationDecorator(options.EvmKeeper),
+		ethante.NewEthAccountVerificationDecorator(options.AccountKeeper, options.EvmKeeper),
+		ethante.NewCanTransferDecorator(options.EvmKeeper),
+		ethante.NewVirtualFrontierContractDecorator(options.EvmKeeper), // prevent transfer to virtual frontier contract
```

Comment:
`NewVirtualFrontierContractDecorator` can be removed from rollapp-evm 
`vfc` is not wired in `app.go`

Reply:
ok, just FYI though
it's here as well https://github.com/dymensionxyz/ethermint/blob/b0f9c1b0a37f6fd06d9dc61745ec1e9561461a29/app/ante/handler_options.go#L78
which is what we would depend on if we didn't implement our own